Core/Wrappers/Damages/Damage.cs

```diff
@@ -473,18 +574,34 @@ private static double CalculatePhysicalDamage(this Obj_AI_Base source, Obj_AI_Ba
                 // + Exhausts target enemy champion, reducing their Movement Speed and Attack Speed by 30%, their Armor and Magic Resist by 10, and their damage dealt by 40% for 2.5 seconds.
                 if (source.HasBuff("Exhaust"))
                 {
-                    amount /= 0.4d;
+                    amount *= 0.6d;
                 }
             }
 
             var targetHero = target as Obj_AI_Hero;
             if (targetHero != null)
             {
+                // Masteries:
+
+                // BondofStone
+                // + DAMAGE REDUCTION 4 %, doubling to 8 % when near at least one allied champion
+                // + IN THIS TOGETHER 8 % of the damage that the nearest allied champion would take is dealt to you instead.This can't bring you below 15% health.
+                if (targetHero.GetResolve(DamageMastery.Resolve.BondofStone).IsValid())
+                {
+                    amount *=
```

Comment:
not 1000 range?
http://leagueoflegends.wikia.com/wiki/Bond_of_Stone_(Season_2016_Mastery)